tasks/62_swap-cases/swap-cases.js

```diff
@@ -0,0 +1,6 @@
+export function swapCases(str) {
+  return str.split('').map((char) => {
+    return char === char.toUpperCase()?
+       char.toLowerCase() : char.toUpperCase();
+  }).join('');
```

Comment:
How about using `replace()` instead of `split()` and `join()` where the first argument should be regex and the second callback function.

Reply:
Great idea, I changed the function to use `replace()`